app/controllers/admin/tutorials_controller.rb

```diff
@@ -5,12 +5,16 @@ def edit
 
   def create
     tutorial = Tutorial.create(new_tutorial_params)
-    video = tutorial.videos.create(new_video_params)
-    if tutorial.save && video.save
-      redirect_to "/tutorials/#{tutorial.id}"
+    if tutorial.save
+      video = tutorial.videos.create(new_video_params)
+      if tutorial.save && video.save
```

Comment:
Since we no longer require a video to be created, we don't really need this check anymore.